java/integTest/src/test/java/glide/SharedCommandTests.java

```diff
@@ -5890,4 +5892,104 @@ public void test_dump_restore_withOptions(BaseClient client) {
                                         .get());
         assertInstanceOf(RequestException.class, executionException.getCause());
     }
+
+    @SneakyThrows
+    @ParameterizedTest(autoCloseArguments = false)
+    @MethodSource("getClients")
+    public void sscan(BaseClient client) {
+        String key1 = "{key}-1" + UUID.randomUUID();
+        String key2 = "{key}-2" + UUID.randomUUID();
+        long initialCursor = 0;
+        long defaultCount = 10;
+        String[] numberMembers = new String[125];
+        for (int i = 0; i < numberMembers.length; i++) {
+            numberMembers[i] = String.valueOf(i);
+        }
+        String[] charMembers = new String[] {"a", "b", "c", "d", "e"};
+        int resultCursorIndex = 0;
+        int resultCollectionIndex = 1;
+
```

Comment:
try to iterate all set with small page size in a loop

Reply:
Unfortunately, this is not possible as the result set is not fetch in the same sequence as when it was added. I just pushed [7a1cc07](https://github.com/Bit-Quill/glide-for-redis/pull/394/commits/7a1cc07aa2ee277dd3e91165da924bc07e8faeff) to fix the flaky results.

Reply:
You can store fetched results in a `Set` and then compare with the input in order-agnostic manner